File: 1957-delete-characters-to-make-fancy-string/1957-delete-characters-to-make-fancy-string.java
class Solution {
    public String makeFancyString(String s) {
        int count=1;
        StringBuilder sb = new StringBuilder();
        sb.append(s.charAt(0));
        
        for(int i=1; i<s.length(); i++)
        {
            if(s.charAt(i-1) == s.charAt(i))
            {
                count++;
            }
            else
            {
                count=1;
            }
            if(count<3)
            {
                sb.append(s.charAt(i));
            }
        }
        return sb.toString();
    }
}

File: 1957-delete-characters-to-make-fancy-string/1957-delete-characters-to-make-fancy-string.cpp
class Solution {
public:
    string makeFancyString(string s) {
        
        int count=1;
        string result="";
        result.push_back(s[0]);
        
        for(int i=1; i<s.length(); i++)
        {
            if(s[i]==s[i-1])
            {
                count++;
            }
            else
            {
                count=1;
            }
            if(count<3)
            {
                result.push_back(s[i]);
            }
        }
        return result;
    }
};